compiler/rustc_const_eval/src/transform/validate.rs

```diff
@@ -285,6 +285,12 @@ impl<'a, 'tcx> CfgChecker<'a, 'tcx> {
             UnwindAction::Unreachable | UnwindAction::Terminate(UnwindTerminateReason::Abi) => (),
         }
     }
+
+    fn is_critical_edge(&self, target: Option<BasicBlock>, unwind: UnwindAction) -> bool {
```

Comment:
Could you rename or inline this method? It is specific to *call* edges, and does not detect critical edges due to a switchint.